packages/veritone-react-common/src/components/SourceTypeField/styles.scss

```diff
@@ -0,0 +1,19 @@
+.textFieldLabel {
+  font-weight: normal !important;
+}
+
+.dateTimeContainer {
+  padding-top: 18px !important;
+}
+
+.dateTimeLabel {
+  transform: translate(0, 1.5px) scale(0.75) !important;
+  transform-origin: top left !important;
```

Comment:
make sure you check the customization available thru props before doing weird hacks like this

https://material-ui.com/api/input-label/
`<InputLabel shrink ...`

Reply:
not my code but fixed